olcPGEX_AdditionalColours.h

```diff
@@ -6,13 +6,81 @@ namespace olc
 {
 	namespace Colours
 	{
+		//ALL THE COLORS OF THE WIND!
 		static const Pixel
+			BLACK(24, 24, 26),
+			WHITE(239, 248, 254),
+
+			GREY(211, 231, 232),
+			DARK_GREY(123, 135, 143),
+			VERY_DARK_GREY(58, 61, 69),
+
+			RED(230, 65, 48),
+			DARK_RED(193, 14, 0),
+			VERY_DARK_RED(77, 18, 13),
+
 			ORANGE(255, 140, 0),
-			DARK_ORANGE(255, 127, 0),
-			VERY_DARK_ORANGE(255, 114, 0),
+			DARK_ORANGE(176, 100, 0),
+			VERY_DARK_ORANGE(112, 51, 0),
+
+			YELLOW(255, 241, 88),
+			DARK_YELLOW(235, 206, 0),
+			VERY_DARK_YELLOW(117, 99, 0),
+
+			APPLE_GREEN(206, 255, 89),
+			DARK_APPLE_GREEN(174, 222, 0),
+			VERY_DARK_APPLE_GREEN(84, 107, 0),
+
+			GREEN(33, 255, 0),
+			DARK_GREEN(66, 166, 0),
+			VERY_DARK_GREEN(28, 69, 0),
+
+			CYAN(119, 255, 242),
+			DARK_CYAN(0, 214, 168),
+			VERY_DARK_CYAN(0, 97, 61),
+
+			COBALT_BLUE(0, 160, 255),
+			DARK_COBALT_BLUE(0, 97, 176),
+			VERY_DARK_COBALT_BLUE(28, 56, 92),
+
+			BLUE(21, 0, 255),
+			DARK_BLUE(34, 0, 158),
+			VERY_DARK_BLUE(22, 0, 69),
+
+			VIOLET(145, 61, 255),
```

Comment:
The violet should probably be purple, then the other two a 'grade' down, so we maintain consistency.